READMEs/install.md

```diff
@@ -44,12 +44,20 @@ pip3 install --pre ocean-lib
 #### Create your network configuration file
 
 ocean.py uses brownie to connect to deployed smart contracts. To configure the RPC URLs, gas prices and other settings,
```

Comment:
Comment from Veronica: 
Intimidating language for the lay user: “To configure the RPC URLs, gas prices and other settings,... ------> 
'Please check that you have configured all networks before proceeding.'